libs/auth/env_test.go

```diff
@@ -79,3 +79,40 @@ func TestGetEnvFor(t *testing.T) {
 	assert.False(t, ok)
 	assert.Empty(t, out)
 }
+
+func TestAuthEnvVars(t *testing.T) {
+	expected := []string{
```

Comment:
This test can break on SDK bumps. I don't think this breakage adds value.

Can you test instead that the list is non-empty and includes a few common entries?